dbclient/dbclient.py

```diff
@@ -96,68 +118,104 @@ def delete_dir_if_empty(local_dir):
         if len(os.listdir(local_dir)) == 0:
             os.rmdir(local_dir)
 
+    def _loop_until_token_renewed(self):
+        interval = 120
+        timeout = 86400
+
+        for x in range(0, int(timeout / interval)):
+            print(f"#{x} Migration paused due to expired token. Trying to renew...")
+            login_args = parser.get_login_credentials(profile=self._profile)
+            url = login_args['host']
+            token = login_args['token']
+            if token == self._raw_token:
+                print("No new token found. Please renew token by running:\n" +
+                      f"$ databricks configure --token --profile {self._profile}\n" +
+                      f"Will try again in {interval} seconds.")
+                time.sleep(interval)
```

Comment:
Is it possible to make it user-interactive, instead of making it wait for 120 seconds? Having to wait for 2 minutes every time the token expires will be very annoying.

Several ways I can think of:

1. Let the user enter the new DB token, and we just inject it into the self.token (without changing databrickscfg)
2. Let the user enter the new DB token, and we fix the databrickscfg for the user.
3. Let the user fix the databrickscfg, but once finished let the user enter yes or something to the command to resume the process

Reply:
Decoupling token configuration and migration tool makes more sense to me because users will use the tool they are already familiar with for token configuration.

120s is just one random config. If the interval is the only concern here, we can change it to 10s.

Reply:
Yeah, this is fine as is. Reducing 120s to some smaller seconds like 20s would be enough

Reply:
Done.